app/experimenter/experiments/api/v5/types.py

```diff
@@ -429,54 +562,31 @@ def resolve_ready_for_review(self, info):
             ready=ready,
         )
 
-    def resolve_targeting_config_slug(self, info):
+    def resolve_targetingConfigSlug(self, info):
         if self.targeting_config_slug in self.TargetingConfig:
             return self.TargetingConfig(self.targeting_config_slug).value
         return self.targeting_config_slug
 
-    def resolve_targeting_config(self, info):
-
+    def resolve_targetingConfig(self, info):
+        targeting_config = NimbusExperiment.TARGETING_CONFIGS[self.targeting_config_slug]
```

Comment:
oh that's a great change, I mean its a small change, but it is for sure more readable and avoids re-reading configs again and again